pages/linux/laptop-detect.md

```diff
@@ -0,0 +1,16 @@
+# laptop-detect
+
+> Attempts to determine if it's being run on a laptop or a desktop.
```

Comment:
I think imperative mood sounds better
```suggestion
> Attempt to determine if this script is being run on a laptop or a desktop.
```

Reply:
Agree on imperative mood, but thoughts on swapping `this` for `the`?

```suggestion
> Attempt to determine if the script is being run on a laptop or a desktop.
```

Could also consider swapping `is being run on` to `is running on`?

```suggestion
> Attempt to determine if the script is running on a laptop or a desktop.
```

We could also replace `a laptop or a desktop` with just `a laptop or desktop`.

```suggestion
> Attempt to determine if the script is running on a laptop or desktop.
```

Feel free to say which ever you like most (or a combination of those changes) and I can commit it.

Reply:
For now, I've committed the last suggestion. (my personal favorite)

I'll keep this unresolved for anyone that wants to read it and chime in though, otherwise what we have now is probably cool.